update Lido frontend app

mymphe opened the pull request:
This PR includes changes to the Lido frontend application. Among those,
- controls for EL Rewards Withdrawal Limit
- controls for EL Rewards Vault
- display staking limit state

Additionally, the app was restructured and refactored into a more readable and maintainable state.

<img width="807" alt="image" src="https://user-images.githubusercontent.com/39704351/171793257-c178dada-6edb-4ac5-822d-979622324b5a.png">


P.S. Sorry for a single commit change, too many moving parts when setting up a local dev environment.

TheDZhon commented on apps/lido/app/src/components/StakingLimitState.js:
```diff
@@ -0,0 +1,34 @@
+      </ListItem>
+      <ListItem label="Restoration rate">
+        <LoadableElement value={stakingLimitInfo?.maxStakeLimitGrowthBlocks}>
+          {stakingLimitInfo?.maxStakeLimitGrowthBlocks} blocks
```
Maybe it's better to show this one as rate (ETH per block), i.e. `maxStakeLimit / maxStakeLimitGrowthBlocks`

TheDZhon commented on apps/lido/app/src/components/state/ElRewardsVault.js:
```diff
@@ -0,0 +1,98 @@
+const validationSchema = yup.object().shape({
+  vault: yup
+    .string('Vault must be a string.')
+    .test(fieldName, 'Vault  must be a valid address.', (vault) => {
```
extra space

TheDZhon commented on apps/lido/app/src/components/state/ElRewardsVault.js:
```diff
@@ -0,0 +1,98 @@
+      </Controls>
+      <SidePanel
+        opened={sidePanelOpen}
+        title="Change Withdrawal Credentials"
```
Not sure that this is correct for the `ELRewardsVault` :thinking:

TheDZhon commented on apps/lido/arapp.json:
```diff
@@ -15,6 +30,11 @@
     },
+    {
+      "name": "Manage protocol contracts role",
+      "id": "MANAGE_PROTOCOL_CONTRACTS_ROLE",
```
Maybe we should remove the obsolete roles?
* `SET_ORACLE`
* `SET_TREASURY`
* `SET_INSURANCE_FUND`

TheDZhon reviewed (changes requested):
Everything is great :+1: 
I propose to address some minor concerns and suggestions. :pray:

mymphe commented:
**Update**
1. Addressed all of the suggestions above.

2. Added tooltips for staking parameters
- Limit set - `Staking limit is the total amount of ether that can be staked in a given block. The limit goes down with each deposit but is passively restored on each block.`
- Max limit - `Hard cap on staking limit, i.e. staking limit will not be restored past this limit.`
- Restoration - `Rate at which the staking limit is passively restored on each block up until the max limit if no deposits are submitted.`
@TheDZhon @ujenjt Please feel free to make suggestions

3. Adjusted main vs secondary panel width ratios to accommodate staking parameters, i.e. Lido state panel is a little narrow, and Beacon stats and Staking status panels are a bit wider, see pic below

4. Script optimization - fetch all Lido state parameters in parallel to decrease initial load time


<img width="426" alt="image" src="https://user-images.githubusercontent.com/39704351/172793047-a379525c-d185-4cc4-ad09-784cacb61c6c.png">

TheDZhon reviewed (approved):
Awesome :+1: 

Maybe change `no deposits are submitted` to `no user funds submitted`.
There is an ambiguity with the `deposits` term :thinking:
